### PR TITLE
Planar detector tests

### PR DESCRIPTION
**Pull request by paullewis**

### CODE REVIEW

**Comment by paullewis** [comment]
Moar tests. Sorry. (Not sorry.)

File: src/detectors/planar-image/planar-image_test.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+const imageData = loadImageAsImageData();
+
+describe('Planar Image Detector', () => {
+  afterEach(() => {
+    reset();
+  });
+
+  it('adds, removes, and detects images', async () => {
+    const data = await loadDataFile();
+    const src = { id: 'Lighthouse', media: [] };
```
Comment (mmocny) [inline src/detectors/planar-image/planar-image_test.ts]: This looks like the `DetectableImage` interface... but if `media` is empty, then I assume it's not actually used in `addDetectionTarget`?  This makes for a confusing API.  Should addDetectionTarget only accept image ID if it directly accepts image data?

(Either way, this seems like deep in the bowels of the detector code, so any API is fine as long as it makes sense to you.. ;)

**Comment by paullewis** [comment]
> Is this an alternative to PR #118 ?

No, the first one dealt with the the nuts and bolts of the wasm (with a little JS glue), and this is more from the PT side where we expose the actual API that we use, and which ultimately goes down to the wasm stuff in 118.

**Comment by paullewis** [comment]
Merging because of the flaky test outlined in #124

**Pull request merged**